### PR TITLE
[beast] lint and type updates

### PR DESCRIPTION
**Pull request by chejennifer**
- add linting and fix current lint errors
- update CsvData type

### CODE REVIEW

**Review by jehangiramjad — APPROVED** [review]

**Review by shifucun — COMMENTED** [review]
Maybe add a new cloudbuild file in https://github.com/datacommonsorg/import/tree/master/build to enforce the test and lint pass

**Comment by chejennifer** [comment]
> Maybe add a new cloudbuild file in https://github.com/datacommonsorg/import/tree/master/build to enforce the test and lint pass

done

File: build/cloudbuild.npm.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This cloud build is triggered when there is code change in the wizard folder.
+# Run npm tests and lint
+
+# Run Python tests, lint, etc.
+steps:
+  - id: npm_test
+    name: npm:3.7
```
Comment (shifucun) [inline build/cloudbuild.npm.yaml:22]: should name be "node:16"?

You can test this by running `cloud-build-local --config=build/cloudbuild.npm.yaml --dryrun=false .`

File: wizard/src/types.ts [hunk 2]
```diff
@@ -69,24 +70,29 @@ export interface CsvData {
...
   // the orderedColumnNames.
-  rowsForDisplay: Map<BigInt, Array<string>>,
-}
+  rowsForDisplay: Map<BigInt, Array<string>>;
+
+  // if csv input was a user uploaded file, the uploaded csv file.
+  rawCsvFile?: File;
 
+  // if csv input was a user entered url, the url to get the csv file.
+  rawCsvUrl?: string;
```
Comment (pradh) [inline wizard/src/types.ts:87]: nit: one of these must be set?

**Review by pradh — APPROVED** [review]
Nice!  Thanks for adding this.

**Review by shifucun — APPROVED** [review]

File: wizard/src/types.ts (hunk 2 above)
Comment (chejennifer) [reply wizard/src/types.ts:87]: I don't think there's a way to enforce that one of these is set, but I can add a comment

Comment (pradh) [reply wizard/src/types.ts:87]: Yup, comment is good.

File: build/cloudbuild.npm.yaml (hunk 1 above)
Comment (chejennifer) [reply build/cloudbuild.npm.yaml]: you're right, fixed!

File: wizard/src/types.ts (hunk 2 above)
Comment (chejennifer) [reply wizard/src/types.ts:89]: done

**Pull request merged**